test/lambda/api/index.test.ts

```diff
@@ -0,0 +1,29 @@
+import { APIGatewayProxyEventV2, Context, Callback } from "aws-lambda";
+import { indexHandler } from  '../../../lib/lambda/api/index'
+
+describe('Unit test for app handler', function () {
```

Comment:
```suggestion
describe('Unit test for app handler', () => {
```
Shorthand is more concise.